oecophylla/taxonomy/taxonomy.rule

```diff
@@ -328,4 +433,5 @@ rule taxonomy:
     input:
         taxonomy_dir + "metaphlan2/combined_profile.biom",
         taxonomy_dir + "kraken/combined_profile.biom",
+        taxonomy_dir + "centrifuge/combined_profile.biom",
         taxonomy_dir + "shogun/combined_profile.biom"
```

Comment:
similar to above comment -- calling from `rules.[rule_name].output` simplifies maintenance and updating.